lib/vmdb/deprecation.rb

```diff
@@ -23,13 +18,23 @@ class << self
 
     private
```

Comment:
I know it's not needed since they are all class methods, but I added it to visually separate the public and private methods.